dialogTemplate/de.json

```diff
@@ -229,41 +187,35 @@
             "matchingStrictness": null,
             "automaticallyExtensible": true,
             "useSynonyms": true,
-            "values": [
-                {
-                    "value": "remaining",
-                    "synonyms": [
-						"wie lange",
-						"zeit übrig",
-						"übrig",
-						"verbleiben",
-						"rest",
-						"wie viel",
-						"wie lange",
-						"verbleibt"
+			"values": [
+				{
+					"value": "remaining",
+					"synonyms": [
+						"Zeit linke",
+						"wie lange"
 					]
-                }
-            ]
-        }
+				}
+			]
+		}
 	],
-    "intents": [
+	"intents": [
 		{
 			"name": "SetUpAlarm",
 			"enabledByDefault": true,
 			"utterances": [
-				"Schreibe einen {Alarm:=>AlarmEvent} für {1 Stunde:=>Duration}",
-				"Setze einen {alarm:=>AlarmEvent} für {1 Minute:=>Duration}",
-				"Kannst du bitte einen {Alarm:=>AlarmEvent} für {3 Stunden:=>Duration} erstellen",
-				"Setze einen {alarm:=>AlarmEvent} in {10 Minuten:=>Duration}",
-				"Erstelle einen {Alarm:=>AlarmEvent} für {20 Minuten:=>Duration} bitte",
-				"Setze please in {15 minuten:=>Duration} einen {alarm:=>AlarmEvent}.",
-				"Bitte setze einen {alarm:=>AlarmEvent} für {Dienstag um 3 Uhr:=>AlarmDateAndTime}",
-				"Bitte erstelle den {alarm:=>AlarmEvent} in {10 minutes:=>AlarmDateAndTime}",
-				"Erstell einen {20 Minuten:=>AlarmDateAndTime} {alarm:=>AlarmEvent}",
-				"Erstelle einen {alarm:=>AlarmEvent} um {2 Uhr:=>AlarmDateAndTime}",
-				"Füge einen {alarm:=>AlarmEvent} für {4 Uhr:=>AlarmDateAndTime} hinzu.",
-				"{wecke mich:=>AlarmEvent} in {20 minuten:=>Duration} bitte",
-				"Bitte {wake me:=>AlarmEvent} in {20 minuten:=>Duration}"
+				"Stellen Sie einen {Alarm:=>AlarmEvent} von {1 Stunde:=>Duration}",
+				"Stellen Sie einen {Alarm:=>AlarmEvent} von {1 Minute:=>Duration}",
+				"Können Sie eine {Alarm:=>AlarmEvent} für {3 Stunden:=>Duration} erstellen Sie bitte",
+				"Set a {Alarm:=>AlarmEvent} in {10 Minuten:=>Duration}",
+				"erstellen {Alarm:=>AlarmEvent} für {20 Minuten:=>Duration} bitte",
```

Comment:
translation can't use the right conjugation when there are slots right behind - it just uses the basic form